packages/components/src/confirm-dialog/test/index.js

```diff
@@ -29,6 +29,34 @@ describe( 'Confirm', () => {
 
 				expect( dialog ).toBeInTheDocument();
 
+				elementsTexts.forEach( ( txt ) => {
+					const el = wrapper.getByText( txt );
+					expect( el ).toBeInTheDocument();
+				} );
+			} );
+			it( 'should render correctly with custom button lables', () => {
+				const cancelButtonText = 'No thanks';
+				const confirmButtonText = 'Yes please!';
+				const wrapper = render(
```

Comment:
Non-blocking: I see this "wrapper" pattern is taken from other tests in the file, but dropping this link in case you want your tests to be more up-to-date 🙂
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
Much appreciated! I read up a bit on `screen` and implemented it in the new test (f43500b). I can do the same for other, similar `wrappers` in the file as well... unless you think that would be better left for a separate PR?

Reply:
Cool! No strong opinion either way, but personally I don't think the `wrapper` thing is worth going in _just_ to fix it. More like a "fix it if you touch it" thing.

Reply:
Sounds good! Thanks again!